### PR TITLE
Add Murmur VoIP server

### PR DESCRIPTION
**Pull request by dylanmtaylor**
# Pull request

**Purpose**
Murmur is the server for the Mumble voice-over-ip client.

**Approach**
This adds Murmur, a new application 

**Requirements**
Check all boxes as they are completed

- [x] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by dylanmtaylor** [comment]
So this actually works as-is, but the only issue is the 'SuperUser' password is generated on the first run and you have to get it using the docker logs command and finding it in the output.

**Comment by dylanmtaylor** [comment]
I documented the issue, and explained how to get it working. I believe this pull request is ready to be merged now.

**Comment by dylanmtaylor** [comment]
Actually, the documentation should probably suggest using an override file to set the password instead of modifying the file included with DockSTARTer

**Comment by nemchik** [comment]
I think we can include the extra variables and just leave them blank by default, or set them to whatever default is documented.

**Comment by nemchik** [comment]
See bitwarden, watchtower, or Ouroboros for examples. Lots of extra vars, but left blank. The user can set them if they need them and not need an override.

**Comment by dylanmtaylor** [comment]
I've done a bit of testing with this, and I'm happy with it. I was able to connect over LAN (some sample log output):

```
<W>2020-08-25 15:41:08.830 1 => Password for 'SuperUser' set to 'BRy2NegTS3NV'
<W>2020-08-25 15:41:08.863 1 => Server listening on 0.0.0.0:64738
<W>2020-08-25 15:41:08.871 1 => Failed to set IPV6_RECVPKTINFO for 0.0.0.0:64738
<W>2020-08-25 15:41:08.933 1 => Generating new server certificate.
<W>2020-08-25 15:41:09.009 1 => Announcing server via bonjour
<W>2020-08-25 15:41:09.017 1 => Not registering server as public
<W>2020-08-25 15:43:33.028 1 => <1:(-1)> New connection: 172.18.0.1:55638
<W>2020-08-25 15:43:33.049 1 => <1:(-1)> Connection closed: The remote host closed the connection [1]
<W>2020-08-25 15:43:34.459 1 => <2:(-1)> New connection: 172.18.0.1:55642
<W>2020-08-25 15:43:34.474 1 => <2:(-1)> Client version 1.3.2 (X11: 1.3.2)
<W>2020-08-25 15:43:34.478 1 => Starting voice thread
<W>2020-08-25 15:43:34.483 1 => CELT codec switch ffffffff80000010 0 (prefer ffffffff80000010) (Opus 1)
<W>2020-08-25 15:43:34.488 1 => <2:dt(-1)> Authenticated
<W>2020-08-25 15:45:23.634 1 => <2:dt(-1)> Connection closed: The remote host closed the connection [1]
<W>2020-08-25 15:45:23.642 1 => Ending voice thread
<W>2020-08-25 15:45:27.518 1 => <3:(-1)> New connection: 172.18.0.1:55654
<W>2020-08-25 15:45:27.532 1 => <3:(-1)> Client version 1.3.2 (X11: 1.3.2)
<W>2020-08-25 15:45:27.538 1 => Starting voice thread
<W>2020-08-25 15:45:27.543 1 => <3:dt(-1)> Authenticated
<W>2020-08-25 15:48:28.057 1 => <4:(-1)> New connection: 192.168.2.206:39832
<W>2020-08-25 15:48:28.127 1 => <4:(-1)> Connection closed: Error during SSL handshake: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown [13]
<W>2020-08-25 15:48:29.514 1 => <5:(-1)> New connection: 192.168.2.206:39834
<W>2020-08-25 15:48:29.741 1 => <5:(-1)> Client version 1.2.5 (Android: Mumla 3.5.5)
<W>2020-08-25 15:48:29.748 1 => CELT codec switch ffffffff8000000b 0 (prefer ffffffff8000000b) (Opus 1)
<W>2020-08-25 15:48:29.756 1 => <5:Mumla_User(-1)> Authenticated
<W>2020-08-25 15:49:03.611 1 => <3:dt(-1)> Connection closed: The remote host closed the connection [1]
<W>2020-08-25 15:49:20.223 1 => <6:(-1)> New connection: 172.18.0.1:55718
<W>2020-08-25 15:49:20.239 1 => <6:(-1)> Client version 1.3.2 (X11: 1.3.2)
<W>2020-08-25 15:49:20.264 1 => <6:SuperUser(0)> Authenticated
```

> I think we can include the extra variables and just leave them blank by default, or set them to whatever default is documented.

The docs say that if the variable is set, Mumble will not start, but will instead set the SuperUser password to the variable value. I will do some quick testing with it set to an empty string though.

**Comment by dylanmtaylor** [comment]
Okay, that seemed to work actually.

WARNING: The MURMUR_SUPW variable is not set. Defaulting to a blank string.
Recreating murmur ... 
Recreating murmur ... done

It's running fine. I'll make an update to add that variable.

**Comment by dylanmtaylor** [comment]
@nemchik - after that change, it still seems to work just fine. Ready for you to merge it in if you're happy with it.

**Comment by dylanmtaylor** [comment]
Would you like me to rebase and squash these commits or should I leave them as-is? I know that GitHub has a "squash and merge" feature. I just realize that there has been a ton of iteration here, and it might be cleaner if I make it all one commit.

File: compose/.apps/murmur/murmur.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+services:
+  murmur:
+    container_name: murmur
+    environment:
+      - PGID=${PGID}
+      - PUID=${PUID}
+      - TZ=${TZ}
+      - MURMUR_SUPW=${MURMUR_SUPW}
```
Comment (nemchik) [inline compose/.apps/murmur/murmur.yml:8]:
```suggestion
      - MURMUR_SUPW=${MURMUR_SUPW}
      - PGID=${PGID}
      - PUID=${PUID}
      - TZ=${TZ}
```
Sorting alphabetically and this is good to go.

**Comment by dylanmtaylor** [comment]
@nemchik I made the change, I agree it does look better in alphabetical order.

**Comment by nemchik** [comment]
Thanks!

**Pull request merged**